devel/dotnet-sdk/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                dotnet-sdk
+version             5.0.202
+revision            0
+categories          devel dotnet
+license             MIT
+maintainers         {tsabirgaliev @tsabirgaliev} openmaintainer
+
+description         Core functionality needed to create .NET Core projects, that is shared between Visual Studio and CLI
+long_description    .NET is a free, cross-platform, open source developer platform \
+                    for building many different types of applications. \
+                    \
+                    With .NET, you can use multiple languages, editors, and libraries \
+                    to build for web, mobile, desktop, games, and IoT.
+
+homepage            https://dotnet.microsoft.com/
+platforms           darwin
+distname            ${name}-${version}-osx-x64
+master_sites        https://download.visualstudio.microsoft.com/download/pr/a7d92db4-5f1f-4478-9caf-6e45c6ac8b09/2363d553132aaed687265c563431673b/
+
+checksums           rmd160  40c6aa6ea582ae4e7d7b7a69f9a74190d067d680 \
+                    sha256  6f270432de6c41d74f8ec19c999ae62189f253dacfd1d095131ddc10bf6d9d52 \
+                    size    135159658
+
+worksrcdir          ${name}-${version}
+extract.mkdir       yes
+
+use_configure       no
+
+build {}
+
+destroot {
+    set dotnet_home ${prefix}/share/dotnet/${name}
+    set target ${destroot}${dotnet_home}
+
+    # Create the target java directory
+    xinstall -m 755 -d ${target}
+
+    # Copy over the needed elements of our directory tree
+    copy ${worksrcpath}/dotnet \
+         ${worksrcpath}/ThirdPartyNotices.txt \
+         ${worksrcpath}/LICENSE.txt \
+         ${worksrcpath}/host \
+         ${worksrcpath}/packs \
+         ${worksrcpath}/sdk \
+         ${worksrcpath}/shared \
+         ${worksrcpath}/templates \
+         ${target}
+
+    # Symlink gradle into the bin directory
+    ln -s ${dotnet_home}/dotnet ${destroot}${prefix}/bin
+}
```

Comment:
At the end you can add a livecheck to keep track of updates with `port livecheck <portname>`:

```tcl
livecheck.url       https://raw.githubusercontent.com/dotnet/core/main/release-notes/5.0/releases.json
livecheck.regex     ${name}-(\[0-9\.\]+)-osx
```